lib/valid_email2/address.rb

```diff
@@ -39,6 +40,10 @@ def valid?
       end
     end
 
+    def tagged?
+      valid? && address.local.split(DEFAULT_RECIPIENT_DELIMITER).length > 1
```

Comment:
Any specific reason to why you are splitting on `DEFAULT_RECIPIENT_DELIMITER` and checking the length here than just checking if the string includes the delimiter with `include?`?

Then it would be something like: `address.local.include?(DEFAULT_RECIPIENT_DELIMITER)` Which imho reads a bit cleaner

Reply:
You're absolutely right.